service/header/service.go

```diff
@@ -32,32 +32,32 @@ func NewHeaderService(exchange Exchange, store Store, pubsub *pubsub.PubSub) *Se
 	}
 }
 
-// Start starts the header Service.
+// Start starts the expected Service.
```

Comment:
Wait, I see a pattern here. Why is every "header" replaced with "expected"?

Reply:
refactoring probably gone wrong...